utils/Utils.h

```diff
@@ -24,6 +24,16 @@
 #ifndef __UTILS_UTILS_H__
 #define __UTILS_UTILS_H__
 
+#ifdef ARM_COMPUTE_NO_EXCEPTIONS
```

Comment:
This file is not used by the library: it's only needed for tests / examples, so you shouldn't need to update it ?

Reply:
Ah okay, so the really only changes are needed to npy.hpp?
It looked like this header is utility used by the framework.. obviously I was mistaken.
I will revert changes to this file.

Reply:
We have a different `arm_compute/core/Utils.h` but I don't think `utils/Utils.h` is used outside of the examples and tests

Reply:
yes, that essentially was cause of my confusion. I can confirm that `utils/Utils.h` is not needed!
I will revert not needed changes.